libpod/image/image.go

```diff
@@ -1075,3 +1076,75 @@ func (i *Image) Comment(ctx context.Context, manifestType string) (string, error
 	}
 	return ociv1Img.History[0].Comment, nil
 }
+
+// Tree gets dependencies of image
+func (i *Image) Tree(ctx context.Context) ([]Image, error) {
+	treeDepth := 1
+	rootImage := i
+
+	//Traverse to the root(base) image
+	parent, err := i.GetParent()
+	if err != nil {
+		return nil, err
+	}
+
+	for parent != nil {
+		rootImage = parent
+		parent, err = parent.GetParent()
+		if err != nil {
+			break
+		}
+		treeDepth++
+	}
+
+	//Print all subtree(peers) to the depth of input image
+	printSubTree(rootImage, 0, "└─ ", treeDepth)
+
+	return nil, err
+}
+
+func printSubTree(root *Image, rootSize uint64, prefix string, treeDepth int) {
```

Comment:
This function should have an error return value, so that I can be properly propagated.